test_conformance/device_execution/enqueue_flags.cpp

```diff
@@ -39,6 +39,8 @@ static const char* enqueue_flags_wait_kernel_simple[] =
     NL, "  size_t lid = get_local_id(0);"
     NL, "  size_t tid = get_global_id(0);"
     NL, ""
+    NL, "  array[index * gs + tid] = array[(index - 1) * gs + tid] + 1;"
```

Comment:
Why was this moved?  The whole point of this test is to ensure the child waits for the parent.

Reply:
done